editoast/src/main.rs

```diff
@@ -81,6 +80,10 @@ async fn main() {
 async fn run() -> Result<(), Box<dyn Error + Send + Sync>> {
     let client = Client::parse();
     let pg_config = client.postgres_config;
+    let create_db_pool = || match pg_config.url() {
+        Ok(url) => Ok(Data::new(get_pool(url, pg_config.pool_size))),
+        Err(e) => Err(e),
+    };
```

Comment:
```suggestion
    let create_db_pool = || Data::new(get_pool(pg_config.url()?, pg_config.pool_size));
```

Reply:
done. 

the linter wants to have the {} for the closure though

```
let create_db_pool = || {
        Ok::<_, Box<dyn Error + Send + Sync>>(Data::new(get_pool(
            pg_config.url()?,
            pg_config.pool_size,
        )))
};
```